src/tests/integration/omneo/profile/delete-profile-date.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, expect, test } from 'vitest'
+import { Omneo } from '../../../../omneo'
+import { ProfileDatesAttribute } from '../../../../types'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import randomString from '../../../lib/string/random'
+
+const omneo = new Omneo({
+  tenant: process.env.OMNEO_TENANT as string,
+  token: process.env.OMNEO_TOKEN as string
+})
+
+const getHandle = () => { return `sdk_unit_test_dates_${randomString(5).toLowerCase()}_${Math.floor(Date.now() / 1000)}` }
+const testProfileID = process.env.OMNEO_TEST_PROFILE_ID as string
+
+describe('Profile Date Delete', () => {
+  test('SDK delete profile date', async () => {
+    const payload: ProfileDatesAttribute = {
+      name: 'Wedding',
+      date: '2024-12-02',
+      handle: getHandle(),
+      is_recurring: true,
+      recurring_schedule: 'Yearly',
+      note: 'test notes for omneo delete',
+      relationship: 'Me',
+      description: 'test description for delete'
+    }
+
+    await simpleOmneoRequest('PUT', `/profiles/${testProfileID}`, {
```

Comment:
this date needs to be cleaned up in the event the deletion fails
Please use this response to populate a FAILED_DATES_HANDLES array

And if the deletion test fails to delete, the `afterAll` should delete all dates in FAILED_DATES_HANDLES

Reply:
updated, thanks for the feedback.